AmethystAPI/src/amethyst/runtime/patchManager/PatchManager.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include <cstdint>
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+#include <vector>
+namespace Amethyst {
+	class PatchManager {
+	public:
+		PatchManager() {}
+        ~PatchManager() { RemoveAllPatches(); }
```

Comment:
This might lead to a bit of confusion in the future, since this is dead code, the runtime doesn't call the dtor itself for hot reloading, and there is never a case where we need to support completely ejecting amethyst runtime. Thoughts?

Reply:
Yeah while i do agree for this case.  
The thought behind that was some mods might want to instance their own `PatchManager` for some reason.  
Also implemented just as a handler for the case in case it happens